### PR TITLE
feat: add lancedb vector storage [COG-176]

### PR DESCRIPTION
**Pull request by borisarzentar**

### CODE REVIEW

File: cognee/api/v1/cognify/cognify.py [hunk 1]
```diff
@@ -84,6 +83,12 @@
         for file_metadata in files:
             with open(file_metadata["file_path"], "rb") as file:
                 try:
+                    document_id = await add_document_node(
+                        graph_client,
+                        parent_node_id = f"DefaultGraphModel__{USER_ID}",
+                        document_metadata = file_metadata,
+                    )
+
```
[the remark this answers is not in the document]
Comment (Vasilije1990) [reply cognee/api/v1/cognify/cognify.py]: @borisarzentar should we listen to the bot here?

File: cognee/modules/data/extraction/knowledge_graph/extract_knowledge_graph.py [hunk 2]
```diff
@@ -19,6 +19,6 @@ def sync_extract_knowledge_graph():
         return (await event_loop.run_in_executor(None, sync_extract_knowledge_graph)).graph
         # return compiled_extract_knowledge_graph(text, question = "").graph
     except Exception as error:
-        logger.error("Error extracting graph from content: %s", error, exc_info = True)
-        
+        # TODO: Log error to Sentry
+
```
[the remark this answers is not in the document]
Comment (Vasilije1990) [reply cognee/modules/data/extraction/knowledge_graph/extract_knowledge_graph.py]: @borisarzentar do we have sentry? should we add it here?

[the remark this answers is not in the document]
Comment (borisarzentar) [reply cognee/modules/data/extraction/knowledge_graph/extract_knowledge_graph.py]: @Vasilije1990 We don't have it. But I'm planning to add some error logging service in the next release.

File: cognee/api/v1/cognify/cognify.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (borisarzentar) [reply cognee/api/v1/cognify/cognify.py]: @coderabbitai We already use context manager when opening the file, not sure what you mean here.

**Pull request merged**